stories/core/Table.stories.tsx

```diff
@@ -210,11 +207,23 @@ export const Selectable: Story<TableProps> = (args) => {
             Header: 'Click',
             width: 100,
             Cell: (props: CellProps<{ name: string; description: string }>) => {
-              const onClick = () => onClickHandler(props);
               return (
-                <a className='iui-anchor' onClick={onClick}>
-                  Click me!
-                </a>
+                <span
```

Comment:
No need for the wrapper. You can add `e.stopPropagation()` in anchor `onClick` handler.

Reply:
The wrapper is to show that it takes up full space of the cell

Reply:
But it seems that you need to wrap with some HTML element that has `e.stopPropagation()` ( at least I did until I tried to remove it).

Reply:
I could change it, but then clicking anywhere outside the link in the cell will trigger row selection

Reply:
I guess it is ok that you don't click on the link and it triggers select.